README.md

```diff
@@ -81,6 +81,23 @@ function MyComponentWithComponent({ myValue }) {
 }
 ```
 
+**Providing `ReactRelayContext`**
+
+```js
+import { ReactRelayContext } from "react-relay";
+import environment from "./environment";
+
+function App() {
+  const contextValue = { environment, variables: {} }
+  return (
+    <ReactRelayContext.Provider value={contextValue}/>
+      <MyComponentWithHook/>
+      <MyComponentWithComponent/>
```

Comment:
`MyComponentWithComponent`? 🤔

Reply:
Used the name of the components in the example but it's a bit weird here.

Renamed to  `ComponentWithMutation` for clarity.